documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager/cadl-project.yaml

```diff
@@ -0,0 +1,38 @@
+parameters:
+  "python-sdk-folder":
+    default: "{cwd}/azure-sdk-for-python/"
+  "java-sdk-folder":
+    default: "{cwd}/azure-sdk-for-java/"
+  "js-sdk-folder":
+    default: "{cwd}/azure-sdk-for-java/"
+  "csharp-sdk-folder":
+    default: "{cwd}/azure-sdk-for-java/"
+  "openapi-folder":
+    default: "{project-root}/../dataplane/"
+  "service-directory-name":
+    default: "contosowidgetmanager"
+emit:
+  - "@azure-tools/cadl-autorest"
+options:
+  "@azure-tools/cadl-autorest":
+    output-file: widgets.json
+    azure-resource-provider-folder: ../../data-plane
```

Comment:
This should use the `{openapi-folder}`

Reply:
`openapi-folder` isn't supported currently.
https://github.com/Azure/cadl-azure/blob/main/packages/cadl-autorest/src/lib.ts#L12